### DISCUSSION TITLE
What are the benefits if I save the notebook to my gdrive?

### DISCUSSION THREAD

**Discussion by gilroff** [TheLastBen/fast-stable-diffusion#1819] [Q&A] [answered]
Will SD start faster? will my automatic ui configs be saved?

**Comment by TheLastBen** [comment, marked as answer]
Saving the notebook file to your gdrive will save your notebook settings, but you will not get the updated notebook if you use the saved one for long periods.

**Reply by gilroff** [reply]
oh thankss
